repos/fdbt-site/src/pages/ticketRepresentation.tsx

```diff
@@ -18,59 +19,57 @@ interface TicketRepresentationProps {
-                                    </label>
-                                </div>
-                                <div className="govuk-radios__item">
-                                    <input
-                                        className={`govuk-radios__input ${
-                                            errors.length > 0 ? 'govuk-input--error' : ''
-                                        } `}
-                                        id="set-of-services"
-                                        name="ticketType"
-                                        type="radio"
-                                        value="multipleServices"
-                                    />
-                                    <label className="govuk-label govuk-radios__label" htmlFor="set-of-services">
-                                        A ticket for a set of services
-                                    </label>
-                                </div>
-                            </div>
-                        </FormElementWrapper>
-                    </fieldset>
-                </div>
-                <input type="submit" value="Continue" id="continue-button" className="govuk-button" />
-            </>
-        </CsrfForm>
-    </TwoThirdsLayout>
-);
+const TicketRepresentation = ({ fareType, errors = [], csrfToken }: TicketRepresentationProps): ReactElement => {
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={errors}>
+            <CsrfForm action="/api/ticketRepresentation" method="post" csrfToken={csrfToken}>
+                <>
+                    <ErrorSummary errors={errors} />
+                    <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                        <fieldset className="govuk-fieldset" aria-describedby="ticket-representation-page-heading">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                                <h1 className="govuk-fieldset__heading" id="ticket-representation-page-heading">
+                                    {`Select a type of ${
+                                        fareType === 'multiOperator' ? 'multi-operator' : 'period'
+                                    } ticket`}
+                                </h1>
+                            </legend>
+                            <FormElementWrapper errors={errors} errorId="geo-zone" errorClass="govuk-radios--errors">
+                                <RadioButtons
+                                    inputName="ticketType"
+                                    options={[
+                                        {
+                                            value: 'geoZone',
+                                            label: 'A ticket within a geographical zone',
+                                            hint: 'Unlimited travel within a geographical zone',
```

Comment:
Unsure on the usage of the term 'unlimited' here, as they are very limited and they impose the limitations themselves on the pages later on (duration for example).

Reply:
Note, my dispute with the term is for every option.